target/linux/ath79/nand/base-files/etc/hotplug.d/ieee80211/10-fix-wifi-mac

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+[ "$ACTION" = add ] || exit
+
+PHYNBR=${DEVPATH##*/phy}
+
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case $board in
+zyxel,nbg6716)
+	# Set mac address for ath9k device
+	[ "$PHYNBR" -eq 1 ] && \
+		macaddr_add "$(mtd_get_mac_ascii u-boot-env ethaddr)" 2 > /sys${DEVPATH}/macaddress
```

Comment:
So, you use another MAC address, too? As always, I'd be happy about an overview of MAC addresses used and where they come from (flash locations, calculation offset, etc...) added to the commit message. (Something like the list here: https://openwrt.org/docs/guide-developer/mac.address#merge_your_data)